churchland_pipeline/reference.py

```diff
@@ -5,9 +5,9 @@
 @schema
 class BrainRegion(dj.Lookup):
     definition = """
-    brain_abbrev : varchar(8) # brain region abbreviation
+    brain_abbrev : varchar(255) # brain region abbreviation
```

Comment:
again, keep primary attributes short. `brain_abbrev` can be reasonably limited to 8-12 characters with a fuller description provided in the secondary attributes.